From 55257ac3ee92143d0d8339faa0272f95a792febb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ch=C3=AD-Thanh=20Christopher=20Nguy=E1=BB=85n?=
 <chithanh@gentoo.org>
Date: Tue, 11 Feb 2014 12:08:07 +0100
Subject: [PATCH] glamor_xv: add missing include

glamor_xv.c uses DamageDamageRegion but does not include xorg/damage.h. This
causes the build to fail if 9b8217f9ef6279fff6628631d18497bed0343ef9 is not
present in the X server.

Bugzilla: https://bugs.gentoo.org/show_bug.cgi?id=499442
---
 src/glamor_xv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/glamor_xv.c b/src/glamor_xv.c
index a89b4cd..b40b6e8 100644
--- a/src/glamor_xv.c
+++ b/src/glamor_xv.c
@@ -37,6 +37,7 @@
 #ifdef GLAMOR_XV
 #include "xf86xv.h"
 #include <X11/extensions/Xv.h>
+#include <xorg/damage.h>
 #include "fourcc.h"
 /* Reference color space transform data */
 typedef struct tagREF_TRANSFORM
-- 
1.8.3.2

